frontend/src/services/getData.js

```diff
@@ -0,0 +1,11 @@
+const baseUrl = 'http://urbanplants.local/plant'
+
+export default async function getData() {
+  try {
+    const response = await fetch(baseUrl)
+    const data = response.json()
+    return data
+  } catch (error) {
+    return { error: 'The server is down! :(' }
```

Comment:
Why you don't write the errors in the console?
`catch (error) {
console.error(error)
}`
Might be helpful if it doesnt work...